packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -39,12 +30,12 @@ export class LocalChannelStorageService implements IChannelStorageService {
         return blob;
     }
 
-    private readSyncInternal(path: string, tree: ITree): IBlob | undefined {
+    private readSyncInternal(path: string, tree: ITree): ArrayBufferLike | undefined {
         for (const entry of tree.entries) {
             switch (entry.type) {
                 case TreeEntry.Blob:
                     if (path === entry.path) {
-                        const blob = entry.value as IBlob;
+                        const blob = entry.value;
```

Comment:
another example of blob is of type string. Should I change the ITreeEntry interface to make blob of type ArrayBufferLike?

Reply:
This one is easier to answer (as all the code is in front of us). It uses ITree as a strating point, and today ITree operates with either utf-8 or base64 stirngs. ITree is used when generating snapshots, and long term we want to change it also to use buffers, but it's much bigger change (or maybe better say - independent change from your work). So we are not going to change it in this PR and this code should work with what it got - existing format.
But this class should not expose it on its public API surface - it exposes IChannelStorageService  which should expose readBlob() that returns array, and possibly readString.

Reply:
Since we already have the tree contents in base64/utf-8 at this location, I think we need a conversion to buffer in this case in order to support readBlob() (until the tree uses buffers).

For expediency, I'm suggesting that Si add a helper function to do this in both driver-utils and common-utils, so that she can immediately use the one in driver-utils until we get a common-utils version bump, and then after the version bump we can switch to using the one in common-utils and delete the one in driver-utils.